src/legacy/ui/ui_bundles/app_entry_template.js

```diff
@@ -26,11 +28,22 @@ export const appEntryTemplate = bundle => `
  * context: ${bundle.getContext()}
  */
 
+// import global polyfills
+import Symbol_observable from 'symbol-observable';
+import 'core-js/stable';
+import 'regenerator-runtime/runtime';
+import 'custom-event-polyfill';
+import 'whatwg-fetch';
+import 'abortcontroller-polyfill';
+import 'childnode-remove-polyfill';
+${apmImport()}
 import { i18n } from '@kbn/i18n';
 import { CoreSystem } from '__kibanaCore__'
 
 const injectedMetadata = JSON.parse(document.querySelector('kbn-injected-metadata').getAttribute('data'));
 
+${apmInit('injectedMetadata.apm')}
```

Comment:
this would become `${apmInit('injectedMetadata.vars.apmConfig')}` as per the https://github.com/elastic/kibana/pull/62484/files#diff-ce5eebb41b6315c38446e8a68a4ca0aeR195